test/fixtures/cli-plugin-array/.gitattributes

```diff
@@ -0,0 +1 @@
+*.html text=auto eol=lf
```

Comment:
No. The test is failing on windows because the output has `\n` while the baseline file has `\r\n`. I added .gitattributes to enforce `\n` in the baseline file even on windows to make the test pass.